opa-manager-operator/testing-yamls/ubuntu.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
Where is this file used?

Reply:
these are example files, maybe I should not have commited. Will remove from next commit